app/src/main/java/com/example/taskyy/data/remote/response_objects/EventResponse.kt

```diff
@@ -0,0 +1,10 @@
+package com.example.taskyy.data.remote.response_objects
+
+data class EventResponse(
+    val id: String,
+    val title: String,
+    val description: String,
+    val time: Long,
+    val remindAt: Long,
+    val isDone: Boolean
+)
```

Comment:
This does not represent an event, make sure to stick to the docs and adjust this, then you also won't get a call adapter error :)